areas/data.go

```diff
@@ -0,0 +1,13 @@
+package areas
+
+// AreaDetails represents a response area model from the areas api
+type AreaDetails struct {
+	Code        string `json:"code,omitempty"`
+	Name        string `json:"name,omitempty"`
+	DateStarted string `json:"date_started,omitempty"`
+	DateEnd     string `json:"date_end,omitempty"`
+	NameWelsh   string `json:"name_welsh,omitempty"`
+	Limit       string `json:"limit,omitempty"`
+	TotalCount  string `json:"total_count,omitempty"`
+	Visible     bool   `json:"visible,omitempty"`
+}
```

Comment:
This has been updated on the areas api:

```suggestion
	Code                 string        `json:"code"`
	Name                string        `json:"name"`
	ValidFrom         string        `json:"date_start"`
	ValidTo              string        `json:"date_end"`
	WelshName      string        `json:"name_welsh"`
	GeometricData []map[string]map[string]interface{} `json:"features"`
	Visible                bool          `json:"visible"`
	AreaType           string
```